src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -38,6 +38,52 @@
 logger = logging.getLogger(__name__)
 
 
+def get_variables_properties_from_submission(
+    submission: Submission, options: Dict[str, Any]
+) -> Dict[str, Any]:
+    variables_properties = {}
+
+    connections = options.get("variables_properties", [])
```

Comment:
```suggestion
    connections = options["variables_properties"]
```

Line 537, we already have this check: `if options.get("variables_properties"):`